### ISSUE TITLE
Section about all of the projects that use sympy

### ISSUE THREAD

**Issue by moorepants** [sympy/sympy-paper#28]
There are a number of projects that use sympy as a library, e.g. web apps, domain specific tools, etc. I think we should highlight the best ones in the paper to show SymPy's reach and utility.

**Comment by asmeurer** [comment]
Which libraries should we include?

**Comment by moorepants** [comment]
Definitely the two or three cool web apps (like the photo based one), Sage, Sagemath Cloud. I'll have to look through the list we've been working on in the website.

**Comment by asmeurer** [comment]
Would SymPy Live and SymPy Gamma go on this list?

**Comment by moorepants** [comment]
Yes, they could for sure.

**Comment by ashutoshsaboo** [comment]
Yes, even I definitely feel SymPy Gamma and SymPy Live deserve a special mention, because they are actually SymPy's main web counterparts to other existing web based CASs like Wolfram Alpha etc. 

I have contributed to SymPy Gamma in particular as well and also solved several issues as well. I will definitely give my best to write about SymPy Live and SymPy Gamma.

Also, do we mention about 3rd party apps that use SymPy? Like Mathics/ and also there was some iOS app also listed in our Google group? @asmeurer

**Comment by moorepants** [comment]
Yes, those are the ones that I am primarily referring to. Mathics, Mathpix, cadabra, etc

**Comment by ashutoshsaboo** [comment]
@moorepants Ohh that seems nice. We can add SymPy Gamma and SymPy Live to the paper, and write briefly about the one's that you mentioned as well.

**Comment by moorepants** [comment]
SymJava was influenced by SymPy: https://github.com/yuemingl/SymJava, might be worth metioning somewhere (just learned about it on the mailing list).
